### PR TITLE
Persisting oauth secret to prevent unnecessary regen

### PR DESCRIPTION
**Pull request by ewolinetz**
and update oauthclient if necessary

Fixes issue seen by Jeff

### CODE REVIEW

File: pkg/k8shandler/curation.go [hunk 1]
```diff
@@ -133,12 +133,12 @@ func createOrUpdateCuratorSecret(logging *logging.ClusterLogging) error {
 		"curator",
 		logging.Namespace,
 		map[string][]byte{
-			"ca":       utils.GetFileContents("/tmp/_working_dir/ca.crt"),
-			"key":      utils.GetFileContents("/tmp/_working_dir/system.logging.curator.key"),
-			"cert":     utils.GetFileContents("/tmp/_working_dir/system.logging.curator.crt"),
-			"ops-ca":   utils.GetFileContents("/tmp/_working_dir/ca.crt"),
-			"ops-key":  utils.GetFileContents("/tmp/_working_dir/system.logging.curator.key"),
-			"ops-cert": utils.GetFileContents("/tmp/_working_dir/system.logging.curator.crt"),
+			"ca":       utils.GetFileContents(utils.GetFilePath("ca.crt")),
```
Comment (jcantrill) [inline pkg/k8shandler/curation.go]: Why not abastract this further into a single util method for getting content from a known file location?

Comment (ewolinetz) [reply pkg/k8shandler/curation.go]: Updated

**Comment by jcantrill** [comment]
may resolve https://bugzilla.redhat.com/show_bug.cgi?id=1667796

**Pull request merged**